Add interfaces/secure-contexts.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Note: This file was recently manually updated in commit 03807c2ad6.
      This commit may revert some of those changes.

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/secure-contexts.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389

stephenmcgruer reviewed (changes requested):
@foolip - looks like reffy is still syncing this IDL

foolip commented:
Yes, that was expected, and now this PR serves as the TODO to make it stop :)

autofoolip committed:
Add interfaces/secure-contexts.idl

Note: This file was recently manually updated in commit 03807c2ad6.
      This commit may revert some of those changes.

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/secure-contexts.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389
